### DISCUSSION TITLE
为啥给自己账户转账会失败

### DISCUSSION THREAD

**Discussion by henry652341974** [smartcontractkit/Web3_tutorial_Chinese#9] [Q&A]
我随便写了一个转账。我知道肯定不会成功，但我想知道原因
`

`
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.26;
contract SendDemo {

    function transferMe() public {
        payable(0x64fB697e5e8759B10391c0080F00C070Dd557dA8).transfer(1); 
    }
}

![image](https://github.com/user-attachments/assets/10f48b3e-7035-49f6-9359-ab5f97a5309f)

**Comment by QingyangKong** [comment]
你在调用这个函数之前，有给这个合约转足够的 ETH 进去吗？
